openid_connect_azure_b2c.module

```diff
@@ -13,8 +13,7 @@ function openid_connect_azure_b2c_openid_connect_userinfo_alter(array &$userinfo
   if (!$plugin instanceof OpenIDConnectB2CClient) {
     return;
   }
-
-  // Use the 0th email as the main one
-  $userinfo['email'] = $userinfo['emails'][0];
+  
+  $userinfo['email'] = $plugin->extractEmail($userinfo);
```

Comment:
@Polynya Can I do this now that it's a static method, or should I call `OpenIDConnectB2CClient::extractEmail`?

Reply:
Looks like it works :)